packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -705,6 +720,7 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     } else if ([@"dispose" isEqualToString:call.method]) {
       [_registry unregisterTexture:textureId];
       [_camera close];
+      _dispatchQueue = nil;
```

Comment:
If this line is included. Then the `_dispatchQueue` would need to be recreated when another `CameraController` is initialized.

You can see this problem if you run the example camera app and attempt to switch between cameras.

Reply:
Good catch. Updated to lazy initialize the dispatch queue.

Reply:
I see that you took out passing the `_dispatchQueue` to FLTCamera in commit [7aaccc2](https://github.com/flutter/plugins/pull/1022/commits/7aaccc2f16ae963818a3296efab30af58456ce56).

 I think its causing the error below because `_dispatchQueue` is still null.
```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -
[AVCaptureVideoDataOutput setSampleBufferDelegate:queue:] NULL
sampleBufferCallbackQueue passed'
```

I think you can keep passing `_dispatchQueue` it to `FLTCamera` when `initialize` is called.

Reply:
Updated.